pyvista/__init__.py

```diff
@@ -79,16 +79,11 @@
 else:
     # Set up data directory
     USER_DATA_PATH = appdirs.user_data_dir('pyvista')
-    if not os.path.exists(USER_DATA_PATH):
-        os.makedirs(USER_DATA_PATH)
+    os.makedirs(USER_DATA_PATH, exist_ok=True)
```

Comment:
While we're at it we should probably guard this too with a `try`-`except`, logging failures. Like in the below case.

Reply:
> Good catch, I agree that EAFP is better here than LBYL.
> 
> Out of curiosity: did you get bitten by this, or are you just being cautious? :)

Somewhat unbelievably, I did actually run into this :)

Reply:
Not clear - what error would you like to catch? `os.makedirs(..., exist_ok=True)` already suppresses the `FileExistsError`, which was the only error handled by the previous version of code.

Reply:
I mean the subsequent piece of code that goes
```py
except Exception as e:
    warnings.warn(f'Unable to create `EXAMPLES_PATH` at "{EXAMPLES_PATH}"\n'
                  f'Error: {e}\n\n'
                  'Override the default path by setting the environmental variable '
                  '`PYVISTA_USERDATA_PATH` to a writable path.')
    EXAMPLES_PATH = ''
```